code/style.css

```diff
@@ -1,4 +1,145 @@
-html {
-  background: #0025ff;
-  color: #fff;
+
+.body {
+  background-color: darkred;
 }
+
+
+.page-top-parent {
+
+  border: 10px solid black;
+  width: 100%;
```

Comment:
Please remove `width: 100%` that's making the content of the page overflow (hence the horizontal scrolling)